lib/Cake/View/View.php

```diff
@@ -807,7 +807,7 @@ public function uuid($object, $url) {
  * a layout or other element. Analogous to Controller::set().
  *
  * @param string|array $one A string or an array of data.
- * @param string|array $two Value in case $one is a string (which then works as the key).
+ * @param bool|int|string|array $two Value in case $one is a string (which then works as the key).
```

Comment:
```suggestion
  * @param mixed $two Value in case $one is a string (which then works as the key).
```

Value can be anything. It is also `mixed` in `3.x`

Reply:
Good idea

Reply:
GitHub "Suggested change" somehow added 1 space too much before `*`

Reply:
Fixed